src/libs/actions/Report.js

```diff
@@ -1266,6 +1266,28 @@ function saveReportActionDraft(reportID, reportActionID, draftMessage) {
     Onyx.set(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS_DRAFTS}${reportID}_${reportActionID}`, draftMessage);
 }
 
+/**
+ * Syncs up a chat report and an IOU report in Onyx after an IOU transaction has been made
+ * and they've both been updated in the back-end.
+ * @param {Object} chatReportStuff
```

Comment:
```suggestion
 * and they've both been updated in the back-end.
 
 * @param {Object} chatReportStuff
```
NAB adding one blank line could be beneficial in terms of readability and consistency with the other functions.